### ISSUE TITLE
com.onesignal.InAppMessageView.getDisplayPosition()  on a null object reference 

### ISSUE THREAD

**Issue by mwaked** [OneSignal/OneSignal-Android-SDK#1000]
Hello, 
I received this crash on Firebase, can you please help me to fix it?

**WebViewManager.java line 243
Caused by java.lang.NullPointerException
Attempt to invoke virtual method 'com.onesignal.WebViewManager$Position com.onesignal.InAppMessageView.getDisplayPosition()' on a null object reference
com.onesignal.WebViewManager.calculateHeightAndShowWebViewAfterNewActivity**

<img width="1053" alt="Screen Shot 2020-04-27 at 10 39 32 AM" src="https://user-images.githubusercontent.com/33099498/80346525-8f98e500-8873-11ea-9fdb-c1fa86fa980b.png">

**The code of OneSignal integration**
  
```
 build_one_signal_version = '[0.12.4, 0.99.99]'
 one_signal_version = '3.13.0'
 classpath("gradle.plugin.com.onesignal:onesignal-gradle-plugin:$build_one_signal_version")
 implementation("com.onesignal:OneSignal:$one_signal_version") {
        exclude 'group': 'com.google.android.gms'
 }
```

```
class BaseApplication : Application() {

 override fun onCreate() {
        super.onCreate()
        initOneSignal()
}

 private fun initOneSignal() {
        OneSignal.startInit(this)
                .setNotificationOpenedHandler(NotificationOpenedHandler())
                .setNotificationReceivedHandler(NotificationReceivedHandler())
                .inFocusDisplaying(OneSignal.OSInFocusDisplayOption.Notification)
                .unsubscribeWhenNotificationsAreDisabled(true)
                .init()
    }

}
```

**Comment by rgomezp** [comment]
Howdy,
Can you please include detailed reproduction steps?

**Comment by mwaked** [comment]
```
class NotificationReceivedHandler: OneSignal.NotificationReceivedHandler{
    override fun notificationReceived(notification: OSNotification?) {
    }
}
```
```
class NotificationExtenderService : NotificationExtenderService() {

    override fun onNotificationProcessing(notification: OSNotificationReceivedResult?): Boolean {
 return true
}
}
```
```

class NotificationOpenedHandler : OneSignal.NotificationOpenedHandler {

    override fun notificationOpened(result: OSNotificationOpenResult?) {
}
}
```

**Comment by vgchabelman** [comment]
This is currently my highest crash issue on Crashlytics
```
com.onesignal.WebViewManager.calculateHeightAndShowWebViewAfterNewActivity (WebViewManager.java:243)
com.onesignal.WebViewManager.available (WebViewManager.java:276)
com.onesignal.ActivityLifecycleHandler.setCurActivity (ActivityLifecycleHandler.java:92)
com.onesignal.ActivityLifecycleHandler.onActivityResumed (ActivityLifecycleHandler.java:125)
com.onesignal.ActivityLifecycleListener.onActivityResumed (ActivityLifecycleListener.java:78)
android.app.Application.dispatchActivityResumed (Application.java:455)
android.app.Activity.dispatchActivityResumed (Activity.java:1295)
android.app.Activity.onResume (Activity.java:1827)
android.support.v4.app.FragmentActivity.onResume (FragmentActivity.java:514)
android.app.Instrumentation.callActivityOnResume (Instrumentation.java:1454)
android.app.Activity.performResume (Activity.java:8103)
android.app.ActivityThread.performResumeActivity (ActivityThread.java:4441)
android.app.ActivityThread.handleResumeActivity (ActivityThread.java:4483)
android.app.servertransaction.ResumeActivityItem.execute (ResumeActivityItem.java:52)
android.app.servertransaction.TransactionExecutor.executeLifecycleState (TransactionExecutor.java:176)
android.app.servertransaction.TransactionExecutor.execute (TransactionExecutor.java:97)
android.app.ActivityThread$H.handleMessage (ActivityThread.java:2175)
android.os.Handler.dispatchMessage (Handler.java:107)
android.os.Looper.loop (Looper.java:237)
android.app.ActivityThread.main (ActivityThread.java:7860)
java.lang.reflect.Method.invoke (Method.java)
com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run (RuntimeInit.java:493)
com.android.internal.os.ZygoteInit.main (ZygoteInit.java:1075)
```
Many users report the app crashing on the home screen. I am currently using a recuring in-app message. Some also report the in-app message appearing on every screen they open instead of only once per section.

I am using 
`classpath 'gradle.plugin.com.onesignal:onesignal-gradle-plugin:0.12.6'`
and
`implementation 'com.onesignal:OneSignal:3.13.2'`

And only the general initialization on my Application class.
```        
OneSignal.startInit(this)
    .inFocusDisplaying(OneSignal.OSInFocusDisplayOption.Notification)
    .unsubscribeWhenNotificationsAreDisabled(true)
    .init()
```

**Comment by rgomezp** [comment]
Howdy,
Please upgrade to `3.15.1`. Thanks!

**Comment by vgchabelman** [comment]
Hello,
Update to `3.15.1` last Thursday. So far, we had 155 cases of this issue on this new version. This tracks with previous versions, so I would say it still has not been resolved.

It shows up like this on Crashlytics
`Attempt to invoke virtual method 'com.onesignal.f3$k com.onesignal.u.K()' on a null object reference`

Could ProGuard be affecting it?

**Comment by rgomezp** [comment]
Howdy!
Thanks for your patience. Doesn't look like it had to do with proguard. The [fix](https://github.com/OneSignal/OneSignal-Android-SDK/pull/1119) is merged and will be included in the next release (`3.15.3`) so please keep your eye out for that and make sure to upgrade as soon as it is released.

Enjoy!

**Comment by sujeet-kumar-mehta** [comment]
Hi @rgomezp I am using the OneSignal  implementation 'com.onesignal:OneSignal:[4.0.0, 4.99.99]' but still I am facing this issue. Have you found the fix ?
